container/images/eap8-domain-openjdk21-openshift/run-domain.sh

```diff
@@ -76,53 +70,85 @@ eval preConfigure
 eval configure
 
 imgName=${JBOSS_IMAGE_NAME:-$IMAGE_NAME}
-    imgVersion=${JBOSS_IMAGE_VERSION:-$IMAGE_VERSION}
-
-    log_info "Running $imgName image, version $imgVersion"
-
-    # Handle port offset
-    if [ -n "${PORT_OFFSET}" ]; then
-      PORT_OFFSET_PROPERTY="-Djboss.socket.binding.port-offset=${PORT_OFFSET}"
-    fi
-
-    PUBLIC_IP_ADDRESS=${SERVER_PUBLIC_BIND_ADDRESS:-$(hostname -i)}
-    MANAGEMENT_IP_ADDRESS=${SERVER_MANAGEMENT_BIND_ADDRESS:-0.0.0.0}
-    ENABLE_STATISTICS=${SERVER_ENABLE_STATISTICS:-true}
-
-    #Ensure node name (FOR NOW NEEDED PERHAPS REVISIT FOR EAP8)
-    run_init_node_name
-    #Only in host controller
-    if [ -n "$JBOSS_EAP_DOMAIN_PRIMARY_ADDRESS" ]; then
-      rm -rf /tmp/jvm-cli-script.cli
-      commands="
-        embed-host-controller --std-out=echo --host-config=$JBOSS_EAP_DOMAIN_HOST_CONFIG
-          if (outcome != success) of /host=$JBOSS_EAP_DOMAIN_HOST_NAME/jvm=openshift:read-resource
-            /host=$JBOSS_EAP_DOMAIN_HOST_NAME/jvm=openshift:add"
-          for option in $(echo $PREPEND_JAVA_OPTS); do
-            commands="$commands
-                /host=$JBOSS_EAP_DOMAIN_HOST_NAME/jvm=openshift:add-jvm-option(jvm-option=\"$option\")"
-          done
-      for option in $(echo $JAVA_OPTS); do
-        commands="$commands
-                /host=$JBOSS_EAP_DOMAIN_HOST_NAME/jvm=openshift:add-jvm-option(jvm-option=\"$option\")"
-      done
-      commands="$commands
-         end-if"
-      echo "$commands" >> /tmp/jvm-cli-script.cli
-      cat /tmp/jvm-cli-script.cli
-      $JBOSS_HOME/bin/jboss-cli.sh --file=/tmp/jvm-cli-script.cli
-    fi
-    if [ -n "$JBOSS_EAP_DOMAIN_PRIMARY_ADDRESS" ]; then
-      SERVER_ARGS="-Djboss.domain.primary.address=$JBOSS_EAP_DOMAIN_PRIMARY_ADDRESS $SERVER_ARGS"
-    fi
-    if [ -n "$JBOSS_EAP_DOMAIN_DOMAIN_CONFIG" ]; then
-      SERVER_ARGS="--domain-config=$JBOSS_EAP_DOMAIN_DOMAIN_CONFIG $SERVER_ARGS"
-    fi
-    if [ -n "$JBOSS_EAP_DOMAIN_HOST_CONFIG" ]; then
-      SERVER_ARGS="--host-config=$JBOSS_EAP_DOMAIN_HOST_CONFIG $SERVER_ARGS"
-    fi
-    SERVER_ARGS="${JAVA_PROXY_OPTIONS} -Djboss.node.name=${JBOSS_NODE_NAME} -Djboss.tx.node.id=${JBOSS_TX_NODE_ID} ${PORT_OFFSET_PROPERTY} -b ${PUBLIC_IP_ADDRESS} -bprivate ${PUBLIC_IP_ADDRESS} -bmanagement ${MANAGEMENT_IP_ADDRESS} -Dwildfly.statistics-enabled=${ENABLE_STATISTICS} ${SERVER_ARGS}"
-    $JBOSS_HOME/bin/domain.sh ${SERVER_ARGS} &
+imgVersion=${JBOSS_IMAGE_VERSION:-$IMAGE_VERSION}
```

Comment:
@yersan , this change is only refactoring right (hard to review)?

Reply:
Yes, refactoring mostly. I have not removed anything that we were doing before

Reply:
@jfdenise I realized I forgot to mention I removed the `run_init_node_name`, probably incorrectly. I assumed the node name should be the Host Controller name configured by the users, but maybe that's not correct at all.

I need to review this